pytest-selenium/tests/test_navigation.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from pages.content import Content
+from tests import markers
+
+
+@markers.test_case("C477321")
+@markers.parametrize("page_slug", [("preface")])
+@markers.nondestructive
+def test_previous_link_hidden_on_first_page(selenium, base_url, book_slug, page_slug):
+
+    # GIVEN: The page is loaded
+    content = Content(selenium, base_url, book_slug=book_slug, page_slug=page_slug).open()
+    toolbar = content.toolbar
+    toc = content.sidebar.toc
+
+    # confirm first page is selected
+    if toc.active_section.get_attribute("textContent") != toc.sections[0].section_title:
```

Comment:
can't you use the `section_title` helper on the first part?

Reply:
reorganized the helpers